ghost/core/core/server/services/email-service/email-templates/partials/styles.hbs

```diff
@@ -381,6 +381,17 @@ h6 + .kg-paywall .kg-paywall-hr td {
 }
 
 /* Exclude CTA cards with colored backgrounds from custom text color, but allow transparent ones */
+{{#hasFeature "emailCustomization"}}
+{{#each ctaBgColors}}
+.post-content-row .kg-cta-bg-{{this}} .kg-cta-text p,
+.post-content-row .kg-cta-bg-{{this}} .kg-cta-text ul,
+.post-content-row .kg-cta-bg-{{this}} .kg-cta-text ul li,
+.post-content-row .kg-cta-bg-{{this}} .kg-cta-text ol,
+.post-content-row .kg-cta-bg-{{this}} .kg-cta-text ol li {
+    color: inherit !important;
+}
```

Comment:
Ideally we should be relying on the cascade and inheritance to avoid the need for specific overrides like this but we have too many specific styles to make that feasible without a big rework of the stylesheet.

I've kept the change as minimal as possible so we can fix this bug then we can tackle a larger refactor later if there's appetite.